src/pages/strom/akcie/[[...params]].tsx

```diff
@@ -1,11 +1,9 @@
 import axios from 'axios'
-import clsx from 'clsx'
 import {GetServerSideProps, NextPage} from 'next'
-import Link from 'next/link'
```

Comment:
prestavame tu pouzivat `next/link`, co nemusi nutne vadit, len ma Next nejaku vyhodu ako prefetch stranok a tym padom rychlejsiu navigaciu... tam ale treba rozlisovat, ci su linky lokalne alebo nie, neskor sa da zamysliet, ci na to tiez chceme daky vlastny komponent, alebo ze ako to chceme pouzivat. nesuri